docs/source/api/deepchecks.core.rst

```diff
@@ -18,10 +18,33 @@ deepchecks.core
 .. rubric:: Classes
 
 .. autoclass:: BaseCheck
+    :members:
```

Comment:
In the core - wouldn't it be much more convenient to return previous format - where classes were separate pages & navigatable by sidebar etc.?